grakn-core/src/main/java/ai/grakn/graql/GetQuery.java

```diff
@@ -48,8 +47,9 @@
     /**
      * Get the transaction this query is running against, if it is set
      */
-    @CheckReturnValue
-    Optional<GraknTx> tx();
+    @SuppressWarnings("unsued") //TODO: @Felix, What's the deal here? Seems totally unused
```

Comment:
@aelred

Reply:
Just a getter. Arguably part of the API, but I'm not sure either way. Maybe tag as deprecated?

Reply:
Can't we just kill it? I doubt it's being used externally?

Reply:
I'm good with that! We toe the line of "breaking change", but you're right it's not likely to be used. Plus, we haven't defined our API yet :P